numba/targets/boxing.py

```diff
@@ -338,12 +338,14 @@ def box_smart_array(typ, value, c):
     with c.builder.if_else(cgutils.is_not_null(c.builder, obj)) as (has_parent, otherwise):
         with has_parent:
             c.pyapi.incref(obj)
-            retn = c.pyapi.call_method(obj, "host_changed")
+            host = c.pyapi.string_from_constant_string('host')
```

Comment:
Oh, I see. No, the pyapi module doesn't have unit tests. They would not be very practical to write.